packages/coreutils/stdio-impl.h.patch

```diff
@@ -0,0 +1,11 @@
+--- ../cache/coreutils-8.30/lib/stdio-impl.h	2018-06-24 04:52:06.000000000 +0000
++++ ./lib/stdio-impl.h	2018-07-03 06:31:46.593224857 +0000
+@@ -60,7 +60,7 @@
+ #  define _flags pub._flags
+ #  define _r pub._r
+ #  define _w pub._w
+-# elif defined __ANDROID__ /* Android */
++# elif defined __ANDROID__ && defined __aarch64__ || defined __x86_64__ /* Android */
```

Comment:
Probably doesn't matter for our builds but won't this mean:
If ( def ANDROID && def aarch64 ) || def x86_64
Rather than
If def ANDROID && ( def aarch64 || def x86_64 )
?